include/xsimd/types/xsimd_types_include.hpp

```diff
@@ -67,6 +67,14 @@
 #include "xsimd_neon_complex.hpp"
 #endif
 
+#if XSIMD_ARM_INSTR_SET == XSIMD_VERSION_NUMBER_NOT_AVAILABLE && XSIMD_X86_INSTR_SET == XSIMD_VERSION_NUMBER_NOT_AVAILABLE
+	#if defined(XSIMD_ENABLE_FALLBACK)
+		#warning "No SIMD instructions enabled, using fallback mode."
+	#else
+		#error "Please compile with SIMD instructions enabled or activate the fallback mode. (e.g. for x86 -march=native or for ARM -mfpu=neon)"
```

Comment:
This prevents libraries from including `xsimd.hpp` if they don't enable simd instructions nor fallback (which is the case when testing for instance), I think the warning should be enough.

Reply:
made it a warning